modules/virt-installing-virtio-drivers-installing-windows.adoc

```diff
@@ -1,30 +1,29 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/vm_networking/virt-installing-qemu-guest-agent.adoc
+// * virt/virtual_machines/virt-installing-qemu-guest-agent.adoc
 // * virt/virtual_machines/virt-installing-virtio-drivers-on-new-windows-vm.adoc
 
 [id="virt-installing-virtio-drivers-installing-windows_{context}"]
 = Installing VirtIO drivers during Windows installation
 
-Install the VirtIO drivers from the attached SATA CD driver during Windows installation. 
+Install the VirtIO drivers from the attached SATA CD driver during Windows installation.
 
 [NOTE]
 ====
-This procedure uses a generic approach to the Windows installation and the 
-installation method might differ between versions of Windows. Refer to the 
+This procedure uses a generic approach to the Windows installation and the
+installation method might differ between versions of Windows. Refer to the
 documentation for the version of Windows that you are installing.
 ====
 
 .Procedure
 
 . Start the virtual machine and connect to a graphical console.
-. Begin the Windows installation process. 
+. Begin the Windows installation process.
 . Select the *Advanced* installation.
-. The storage destination will not be recognized until the driver is loaded. 
-Click `Load driver`. 
+. The storage destination will not be recognized until the driver is loaded.
+Click `Load driver`.
 . The drivers are attached as a SATA CD drive. Click *OK* and browse the CD drive
- for the storage driver to load. The drivers are arranged hierarchically 
-according to their driver type, operating system, and CPU architecture. 
-. Repeat the previous two steps for all required drivers. 
-. Complete the Windows installation. 
-
+ for the storage driver to load. The drivers are arranged hierarchically
```

Comment:
I think there's an extra space at the beginning of this line, though that wouldn't affect rendering.